File: DynAPPix.Bastogne.App/TempModels/Organisation.cs
using System;
using System.Collections.Generic;

namespace Dynappix.Bastogne.App.TempModels
{
    public partial class Organisation
    {
        public Organisation()
        {
            User = new HashSet<User>();
        }

        public Guid Id { get; set; }
        public string Name { get; set; }
        public string PrimaryContactName { get; set; }
        public string PrimaryContactEmail { get; set; }

        public virtual ICollection<User> User { get; set; }
    }
}

File: DynAPPix.Bastogne.App/ClientApp/app/models/organisation.ts
export class Organisation {
    public _id: string;
    public name: string;
    public primaryContactName: string;
    public primaryContactEmail: string;

    public constructor(data: any = {}) {
        this.name = data.name || '';
        this.primaryContactName = data.primaryContactName || '';
        this.primaryContactEmail = data.primaryContactEmil || '';
    }
}
